docker-compose/is/dockerfiles/is/Dockerfile

```diff
@@ -17,11 +17,11 @@
 # ------------------------------------------------------------------------
 
 # set base Docker image to WSO2 Identity Server Docker image with latest WSO2 Updates
-FROM docker.wso2.com/wso2is:5.11.0.0
+FROM docker.wso2.com/wso2is:6.0.0.0
```

Comment:
change the license files to `LLC`

Reply:
Addressed with https://github.com/wso2/docker-is/pull/348/commits/675d438b74c80101df0873c45f38a74beeb853b1